src/logos/LogoFacebook.vue

```diff
@@ -5,10 +5,10 @@
     :height="height"
     viewBox="0 0 24 24"
     preserveAspectRatio="xMinYMid"
-    aria-labelledby="logo-facebook"
+    aria-label="logo-facebook"
```

Comment:
@hackmastera According to [this article](https://css-tricks.com/accessible-svgs/), all we need to change is `role="presentation"` to `role="img"`. I can make this change, but since some accessibility recommendations are not normative, can you share what prompted you to make these changes for comparison?

Reply:
@sdellis sorry about that! We didn't see the title at first and so we made it `aria-label`. Then we noticed the title and the typo in the id. Forgot to change back to labelled-by. Thanks for catching it!

So that's fixed now and we went ahead and added the role change that you suggested. Take another look?

Reply:
@hackmastera LGTM! I fixed the role in the other icons as well. Will merge now.